### PR TITLE
Tickets/dm 25150

### PR DESCRIPTION
**Pull request by patrickingraham**
Revs the version to add wavelength focus relationship. Note that only recent configuration files were updated.

### CODE REVIEW

File: ATAOS/v3/hex_m1_hex_202003_v3.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  m2: 500.
+  x: 0.001
+  y: 0.001
+  z: 0.001
+  u: 0.001
+  v: 0.001
+
+chromatic_dependence:
+  - 2.1e-5
+  - 0.0
```
Comment (tribeiro) [inline ATAOS/v3/hex_m1_hex_202003_v3.yaml]: add newline at the end of the file

File: ATAOS/v3/_labels.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Labels for recommended settings; a dict of label: config_file
+current: hex_m1_hex_202003.yaml
```
Comment (tribeiro) [inline ATAOS/v3/_labels.yaml]: I think the names of the files are wrong here should be:

```
current : hex_m1_hex_202003_v3.yaml
constant_hex : hex_m1_202003_constant_hex_v3.yaml
```

They are missing a `_v3`

**Review by tribeiro — CHANGES_REQUESTED** [review]
Please, correct the labels file!

**Comment by patrickingraham** [comment]
Changes made. Also re-based on develop.

**Review by tribeiro — APPROVED** [review]

**Pull request merged**